<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lab12_TNR</class>
 <widget class="QMainWindow" name="Lab12_TNR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab12_TNR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="matr2ResizeBtn_tnr">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>370</y>
      <width>61</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Resize</string>
    </property>
   </widget>
   <widget class="QPushButton" name="matr1ArMean_tnr">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>430</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Find Determinant</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="matr1I_tnr">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>48</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-5</number>
    </property>
    <property name="value">
     <number>4</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="matr2I_tnr">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>330</y>
      <width>48</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-5</number>
    </property>
    <property name="value">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>i:</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>201</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="grid1"/>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>330</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>j:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="matr1J_tnr">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>48</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-5</number>
    </property>
    <property name="value">
     <number>4</number>
    </property>
   </widget>
   <widget class="QPushButton" name="matr1RandomBtn_tnr">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Fill Random</string>
    </property>
   </widget>
   <widget class="QPushButton" name="matr1SetBtn_tnr">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Cahnges</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>191</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="grid3"/>
   </widget>
   <widget class="QComboBox" name="operation_tnr">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>51</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>*</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>^(-1)</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="matr1ResizeBtn_tnr">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>370</y>
      <width>61</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Resize </string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>330</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>j:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="matr2RandomBtn_tnr">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>260</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Fill Random</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="matr2J_tnr">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>330</y>
      <width>48</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-5</number>
    </property>
    <property name="value">
     <number>4</number>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>201</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="grid2"/>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>330</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>i:</string>
    </property>
   </widget>
   <widget class="QLabel" name="matr1Det_tnr">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>430</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="equalsBtn_tnr">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>110</y>
      <width>31</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="matr2SetBtn_tnr">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>220</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Changes</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorMsg_tnr">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>510</y>
      <width>541</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Error message</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="matr1GetValBtn_tnr">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>51</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Value:</string>
    </property>
   </widget>
   <widget class="QLabel" name="matr1Val_tnr">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>370</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="matr2GetValBtn_tnr">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>370</y>
      <width>51</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Value:</string>
    </property>
   </widget>
   <widget class="QLabel" name="matr2Val_tnr">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>370</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="matr2Det_tnr">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>430</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="matr2DetBtn_tnr">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>430</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Find Determinant</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
